doc/release_notes_1.1.rst

```diff
@@ -0,0 +1,514 @@
+- :acrn-issue:`3160` - There is a logic bug when set iommu page walk coherent
+- :acrn-issue:`3181` - [auto][sit][daily]Case "Hypervisor_Launch_RTVM_on_SATA_Storage" sata disk can not passthru
+- :acrn-issue:`3182` - run command "echo c > /proc/sysrq-trigger" can't trigger AaaG warm reboot
+- :acrn-issue:`3184` - fail to locate ACPI RSDP table on some EFI platforms
+- :acrn-issue:`3188` - dm: update uos patch in launch_hard_rt_vm.sh
+
+Known Issues
+************
+
+:acrn-issue:`1773` - USB Mediator: Can't find all devices when multiple USB devices connected
+   After booting UOS with multiple USB devices plugged in,
+   there's a 60% chance that one or more devices are not discovered.
+
+   **Impact:** Cannot use multiple USB devices at same time.
+
+   **Workaround:** Unplug and plug-in the unrecognized device after booting.
+
+-----
+
+:acrn-issue:`1991` - Input not accepted in UART Console for corner case
+   Input is useless in UART Console for a corner case, demonstrated with these steps:
+
+   1) Boot to SOS
+   2) ssh into the SOS.
+   3) use ``./launch_uos.sh`` to boot UOS.
+   4) On the host, use ``minicom -D /dev/ttyUSB0``.
+   5) Use ``sos_console 0`` to launch SOS.
+
+   **Impact:** Fails to use UART for input.
+
+   **Workaround:** Enter other keys before typing :kbd:`Enter`.
+
+-----
+
+:acrn-issue:`2267` - [APLUP2][LaaG] LaaG can't detect 4k monitor
+   After launching UOS on APL UP2 , 4k monitor cannot be detected.
+
+   **Impact:** UOS can't display on a 4k monitor.
+
+   **Workaround:** Use a monitor with less than 4k resolution.
+
+-----
+
+:acrn-issue:`2279` - [APLNUC] After exiting UOS, SOS can't use USB keyboard and mouse
+   After exiting UOS with mediator Usb_KeyBoard and Mouse, SOS cannot use the USB keyboard and mouse.
+
+   These steps reproduce the issue:
+
+   1) Insert USB keyboard and mouse in standard A port (USB3.0 port)
+   2) Boot UOS by sharing the USB keyboard and mouse in cmd line:
+
+      ``-s n,xhci,1-1:1-2:1-3:1-4:2-1:2-2:2-3:2-4 \``
+
+   3) UOS access USB keyboard and mouse.
+   4) Exit UOS.
+   5) SOS tries to access USB keyboard and mouse, and fails.
+
+   **Impact:** SOS cannot use USB keyboard and mouse in such case.
+
+   **Workaround:** Unplug and plug-in the USB keyboard and mouse after exiting UOS.
+
+-----
+
+:acrn-issue:`2753` - UOS cannot resume after suspend by pressing power key
+   UOS cannot resume after suspend by pressing power key
+
+   **Impact:** UOS may failed to resume after suspend by pressing the power key.
+
+   **Workaround:** None
+
+-----
+
+:acrn-issue:`2974` - Launching Zephyr RTOS as a real-time UOS takes too long
+   Launching Zephyr RTOS as a real-time UOS takes too long
+
+   These steps reproduce the issue:
+
+   1) Build Zephyr image by follow the `guide
+      <https://projectacrn.github.io/latest/tutorials/using_zephyr_as_uos.html?highlight=zephyr>`_.
+   2) Copy the "Zephyr.img", "OVMF.fd" and "launch_zephyr.sh" to ISD.
```

Comment:
What is ISD? Probably better to replace this with "platform" or something?

Reply:
Using "NUC" instead. 😄